### PR TITLE
EWPP-1911: Update to PHP7.4/8.0

### PR DESCRIPTION
**Pull request by hernani**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -84,5 +98,14 @@
...
             }
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "composer/installers": true,
+            "drupal/core-composer-scaffold": true,
+            "cweagans/composer-patches": true,
+            "phpro/grumphp": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
```
Comment (nagyad) [inline composer.json:108]: Should be in alphabetic order.

File: composer.json [hunk 2]
```diff
@@ -62,7 +73,10 @@
         }
     },
     "_readme": [
-        "We use the drupaltest/behat-traits feature branch until it gets merged and the package is ready for Drupal 9."
+        "Explicit requirement of behat module due to PHP8 compatibility.",
```
Comment (nagyad) [inline composer.json:76]: Explicit requirement of behat/behat module due to PHP8 compatibility.

File: composer.json [hunk 3]
```diff
@@ -62,7 +73,10 @@
...
+        "Explicit requirement of behat/mink-extension to assure lower compatibility.",
+        "We use the drupaltest/behat-traits feature branch until it gets merged and the package is ready for Drupal 9.",
```
Comment (nagyad) [inline composer.json:78]: This can be removed.

**Review by nagyad — CHANGES_REQUESTED** [review]

**Review by nagyad — DISMISSED** [review]

File: composer.json [hunk 4]
```diff
@@ -6,43 +6,54 @@
...
         "guzzlehttp/guzzle": "~6.3",
-        "instaclick/php-webdriver": "^1.4.7",
         "openeuropa/behat-transformation-context": "~0.1",
-        "openeuropa/code-review": "^1.7",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "openeuropa/code-review": "^2.0",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0-beta5",
+        "phpspec/prophecy-phpunit": "^2",
+        "phpunit/phpunit": "^9.4",
```
Comment (22Alexandra) [inline composer.json:30]: We don't need this. Let's reorder alphabetically.

**Review by 22Alexandra — APPROVED** [review]

**Review by nagyad — DISMISSED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**